app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -77,6 +80,17 @@ class Config {
     return Uint8List.fromList(result.value.codeUnits);
   }
 
+  // GitHub App properties.
+  Future<String> get githubPrivateKey =>
+      _getSingleValue('githubapp_private_pem');
+  Future<String> get githubPublicKey => _getSingleValue('githubapp_public_pem');
+  Future<String> get githubAppId => _getSingleValue('githubapp_id');
+  Future<Map<String, dynamic>> get githubAppInstallations async {
+    final String installations =
+        await _getSingleValue('githubapp_installations');
```

Comment:
I see githubapp_installations is based on your personal repo. Shall we used a more general one? How does this affect the authentication?

Reply:
This is because my personal repo is the only one with the flutter-dashboard installed. Once we add more repos to the aplications we will populating githubapp_installations.

The implementation will look like:

* Checks apis is supported by flutter-dashboard.
* Flutter Dashboard is migrated to the flutter community from my personal account.
* Per repo configurations and installations are added.